### PR TITLE
Update documentation screenshot

### LINKED ISSUE #120: Update documentation screenshots
A bunch of screenshots are out-of-date and need a refresh:

### PR DESCRIPTION
**Pull request by angelozerr**
Update documentation screenshots

Fixes #120

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: docs/Producing.md [hunk 1]
```diff
@@ -27,7 +27,25 @@ The log about produced messages is printed in the `Kafka Producer Log` Output vi
...
-![Screenshot-4](assets/screen-4.png)
+For instance:
+
+```
+--- Produce random messages with built-in faker.js support
+--- Message body simply needs to follow the mustache template syntax
+--- See available fake data at https://github.com/marak/Faker.js/#api-methods
+
+PRODUCER keyed-message
+topic: fakerjs-topic
```
Comment (fbricon) [inline docs/Producing.md:38]: key: dept-{{random.number(5)}}

File: docs/Producing.md [hunk 2]
```diff
@@ -27,7 +27,25 @@ The log about produced messages is printed in the `Kafka Producer Log` Output vi
...
+{
+    "id": "{{random.uuid}}",
```
Comment (fbricon) [inline docs/Producing.md:40]: "dept":"{{random.number(5)}}"

**Review by fbricon — CHANGES_REQUESTED** [review]

File: docs/Producing.md [hunk 3]
```diff
@@ -27,7 +27,25 @@ The log about produced messages is printed in the `Kafka Producer Log` Output vi
 
 Record content can be randomized by injecting mustache-like placeholders of [faker.js properties](https://github.com/Marak/faker.js#api-methods), like ``{{name.lastName}}`` or ``{{random.number}}``. Some randomized properties can be localized via the `kafka.producers.fakerjs.locale` setting.
```
Comment (fbricon) [inline docs/Producing.md:29]: The same seed is used for randomizing the key and the value, so the same randomized content can be injected in both places.

**Comment by fbricon** [comment]
can you change docs/assets/kafka-file-producer-randomized.png so the producer name is random-messages, and we can see the same key / value randomized content, as I previously commented
```
key: dept-{{random.number(5)}}
{
..
"dept":"{{random.number(5)}}"
}
```

File: docs/Producing.md (hunk 1 above)
Comment (angelozerr) [reply docs/Producing.md:40]: fixed

File: docs/Producing.md (hunk 2 above)
Comment (angelozerr) [reply docs/Producing.md:43]: fixed

File: docs/Producing.md (hunk 3 above)
Comment (angelozerr) [reply docs/Producing.md:29]: fixed

**Comment by angelozerr** [comment]
> can you change docs/assets/kafka-file-producer-randomized.png so the producer name is random-messages, and we can see the same key / value randomized content, as I previously commented

done

**Comment by fbricon** [comment]
Thanks @angelozerr!

**Pull request merged**